docker-start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+[ -z "$UID" ] && UID=0
+[ -z "$GID" ] && GID=0
+
+# echo >> /etc/xxx and not adduser/addgroup because adduser/addgroup won't work if uid/gid already exists
+echo -e "xuser:x:${UID}:${GID}:xuser:/home/xuser:/bin/ash\n" >> /etc/passwd
+echo -e "xgroup:x:${GID}:xuser\n" >> /etc/group
+
+# it's better to do that (mkdir and chown) here than in the Dockerfile because it will be executed even on volumes if mounted
+mkdir -p /droppy-data 
+mkdir -p /droppy-data/config
+mkdir -p /droppy-data/files
```

Comment:
Ah, I see. I think I like the other approach better, and doing it all in a single folder can be added as a new command line switch to the application itself.